cpa/tester/test.go

```diff
@@ -25,7 +25,8 @@ func (t Test) NamedCases() []NamedTest {
 	for name, test := range t.Cases {
 		result = append(result, NamedTest{name, test})
 	}
-	slices.SortFunc(result, func(a, b NamedTest) bool { return a.Name < b.Name })
+	sort.Slice(result, func(i, j int) bool { return result[i].Name < result[j].Name })
```

Comment:
there is actually a standard lib `slices` package now, https://pkg.go.dev/slices#SortFunc. We might want to just swap to that dependency? Then we wouldn't need to change this I think? 

I suggest it only because of this snippet from the `sort` package doc:
```
Note: in many situations, the newer [slices.SortFunc](https://pkg.go.dev/slices#SortFunc) function is more ergonomic and runs faster.
```
[doc](https://pkg.go.dev/slices#Sort)

Reply:
Done!

Reply:
great, could you do the same in `cpa/tester/runner.go`?

Reply:
Done!